File: Colleen.c
#include<stdio.h>
/*
	Comment outside the func.
*/
void foo(){}
int main(){
/*
	Comment inside the func.
*/
foo();
char*s="#include<stdio.h>%1$c/*%1$c%2$cComment outside the func.%1$c*/%1$cvoid foo(){}%1$cint main(){%1$c/*%1$c%2$cComment inside the func.%1$c*/%1$cfoo();%1$cchar*s=%3$c%s%3$c;printf(s,10,9,34,s);}%1$c";printf(s,10,9,34,s);}

File: PHP/Colleen.php
<?php
//
//	Comment outside the func.
//
function foo()
{
	//
	//	Comment inside the func.
	//
	$str = "<?php%c//%c//%cComment outside the func.%c//%cfunction foo()%c{%c%c//%c%c//%cComment inside the func.%c%c//%c%c%cstr = %c%s%c;%c%c%cing = sprintf(%cstr,10,10,9,10,10,10,10,9,10,9,9,10,9,10,9,36,34,%cstr,34,10,9,36,36,36,10,9,36,10,10,10,10,9,10,10,10,10);%c%cecho %cing;%c}%cfunction maine()%c{%c%cfoo();%c}%cmaine();%c?>%c";
	$ing = sprintf($str,10,10,9,10,10,10,10,9,10,9,9,10,9,10,9,36,34,$str,34,10,9,36,36,36,10,9,36,10,10,10,10,9,10,10,10,10);
	echo $ing;
}
function maine()
{
	foo();
}
maine();
?>
